cmd/flightctl-api/main.go

```diff
@@ -39,6 +39,12 @@ func main() {
 	}
 	log.Printf("Using config: %s", cfg)
 
+	cfg.Database.Port = uint(8888)
+	cfg.Service.Address = ":3444"
+	cfg.Service.AgentEndpointAddress = ":7445"
+	cfg.Service.AgentGrpcAddress = ":7446"
+	cfg.Queue.AmqpURL = "amqp://user:password@127.0.0.1:5674"
+
```

Comment:
Accidental commit after debugging?

Reply:
Yes, my bad. I just reverted this file.
Thanks Frank